package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "",
+  "version": "",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "@vue/cli": "^4.5.13",
```

Comment:
`@vue/cli` should be in `devDependencies`.

Reply:
Moved.